app/aws/s3.py

```diff
@@ -60,8 +60,18 @@ def get_job_metadata_from_s3(service_id, job_id):
     return obj.get()["Metadata"]
 
 
-def remove_job_from_s3(service_id, job_id):
-    return remove_s3_object(*get_job_location(service_id, job_id))
+# adapted from https://docs.aws.amazon.com/AmazonS3/latest/userguide/example_s3_DeleteObjects_section.html
+def remove_job_batch_from_s3(jobs):
+    bucket = resource("s3").Bucket(current_app.config["CSV_UPLOAD_BUCKET_NAME"])
+    object_keys = [FILE_LOCATION_STRUCTURE.format(job.service_id, job.id) for job in jobs]
+
+    try:
+        response = bucket.delete_objects(Delete={"Objects": [{"Key": key} for key in object_keys]})
+    except botocore.exceptions.ClientError:
+        current_app.logger.exception("Couldn't delete any objects from bucket %s.", bucket.name)
+        raise
```

Comment:
Why log it to raise it? we could just let celery log it itself no? We might end up with two exceptions here per my understanding.